tests/run.py

```diff
@@ -22,14 +29,30 @@
                         fight_style=fight_style, profile=path)
         tests.append(grp)
         for talents in talent_combinations(klass):
-            Test(talents, group=grp, args=[ ('talents', talents) ])
+            Test(talents, group=grp, args=[('talents', talents)])
 
     for fight_style in FIGHT_STYLES:
         grp = TestGroup('{}/{}/covenants'.format(profile, fight_style),
                         fight_style=fight_style, profile=path)
         tests.append(grp)
         for covenant in COVENANTS:
             Test(covenant, group=grp,
-                 args=[ ('covenant', covenant.lower()), ('level', 60) ])
+                 args=[('covenant', covenant.lower()), ('level', 60)])
 
-sys.exit( run(tests) )
+    if not args.no_trinkets:
+        from simc_support.game_data.WowSpec import get_wow_spec_from_combined_simc_name
+        from simc_support.game_data.Trinket import get_trinkets_for_spec
```

Comment:
While I explicitly wanted to avoid external deps (because that was one of the problems with bats imo) I'm ok with it as long as it's optional. What I don't like is that you have to explicitly *disable* the trinket tests to not get yelled at for a missing dependency.

I think these imports should be wrapped into a `try: ... except ImportError:` block and these tests should be implicitly disabled if the import failed, with a suitable message.
But then... I don't really expect anyone to run the tests locally apart from myself.

Reply:
Ok will do.
I do think it is a good thing that they can be run locally, and if it is only to reduce the number of broken commits I have to push.
I definitely wouldn't put too much effort into it, but so far I think we are still keeping with a pretty standard python setup.